README.md

```diff
@@ -89,11 +89,14 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
    #### Quick notes on clouds.yaml
    ```shell
    $ cat clouds.yaml
-   userName: "your-name"
-   apiKey: "your-api-key"
+   clouds:
+    ibmcloud:
+      auth:
+        username: "test@test.com"
```

Comment:
How about using:

```
apiUserName: "Your API Username"
AuthenticationKey: "Your API Authentication Key"
```

@jichenjc @xunpan WDYT?

Reply:
The above proposal also align with the SL UI output.

Reply:
That makes sense. Let me modify to align with SL UI output